### ISSUE TITLE
Peer review for proposal - Group G

### ISSUE THREAD

**Issue by khaukhau** [hknguyen20/COMP4010-TeamA-Project1#2]
1. **Critique on plan:**
* The introduction to the variables is not quite clear, a table for introduction to each variable could be included so that the viewer could interpret each variable better.
* The proposal outlines the use of R for data manipulation and analysis but does not specify which statistical methods or models will be used. For instance, it is important to clarify how the research will handle variables, whether it will use regression analysis, time-series analysis, cluster analysis, etc., and how it plans to address potential confounding factors
2. **Choice of plot:** The objective and the question for analysis are quite straightforward. However, the proposal lacks of choice of plot so that the team could discuss further on this point to get deeper into the dataset.
3. **Feasibility:** In terms of the feasibility of the project, it could be handled effectively but it requires domain knowledge in the music analysis to deliver more insights within this dataset. Therefore, it would be better to conduct research on the domain while answering those questions.
4. **Questions:** The questions are good for a big theme but may be too big to cover within a graph. Therefore the group could consider breaking down the questions into smaller questions and providing visualization for each graph. For example, in the question "how has Taylor Swift's musical style evolved over her career?", the group could break down "musical style" into the genre, or tempo, etc... or define new variables. (note that each question should be answered by 2 figures)

**Comment by tienvu95** [comment]
I agree that all proposals should consider more types of chart. If we are going to use just bar and line charts, I don't think we need to introduce additional chart types thru lectures and AE. PLus they should keep in mind the requirements of using 2 charts for each qn, and ofc, they need a more self-contained intro. Their plan is also ambitious, since considering 10+ variables might result in a presentation without a clear focus

**Comment by nguyenkhoihoang** [comment]
Thank you very much for both your feedbacks. They really help us in revising our proposal. Here is our answer to your comments:

- I agree that it would be more self-contained to directly list out the introduction of variables. We’ve listed them out based on your feedback.

- With regards to plan, we’ll use multiple regression analysis to understand the relationship between dependent variable (album scores) and multiple independent variables (audio features like danceability, etc.). 

- With regards to potential confounding factors, I realize the variables within the dataset are mostly internal factors and so we may need to consider outside factors that could bias our results. For example, the popularity of an album might be influenced by marketing efforts, seasonal effects, or global events. If the variables of the dataset are not enough and  these external factors indeed carry weight, we will include them as external variables in our analysis. 

- We don’t include a definitive choice of plot at this proposal stage because we will need time to explore and experiment with the datasets before deciding the most effective one, but it will be detailed and backed by reasoning in the write-up. 

- While we explore 10+ variables, not all of them may turn out interesting. We only put in potential ones to do analysis but in our final write-up we will only show visualization of most prominent variables.

Once again, greatly appreciate your insights!
